settings.gradle.kts

```diff
@@ -36,7 +36,9 @@ include(
     "sentry-samples:sentry-samples-spring",
     "sentry-samples:sentry-samples-spring-boot",
     "sentry-samples:sentry-samples-spring-boot-webflux",
-    "sentry-samples:sentry-samples-netflix-dgs"
+    "sentry-samples:sentry-samples-netflix-dgs",
```

Comment:
I don't see this project in this PR?

Reply:
I guess it's a diff caused by the new comma